dataRetriever/factory/resolverscontainer/metaResolversContainerFactory.go

```diff
@@ -133,8 +133,15 @@ func (mrcf *metaResolversContainerFactory) AddShardTrieNodeResolvers(container d
 
 	for i := uint32(0); i < shardC.NumberOfShards(); i++ {
 		identifierTrieNodes := factory.AccountTrieNodesTopic + shardC.CommunicationIdentifier(i)
-		resolver, err := mrcf.createTrieNodesResolver(identifierTrieNodes, triesFactory.UserAccountTrie,
-			mrcf.numCrossShardPeers, mrcf.numIntraShardPeers, mrcf.numFullHistoryPeers, mrcf.currentNetworkEpochProvider)
+		resolver, err := mrcf.createTrieNodesResolver(
+			identifierTrieNodes,
+			triesFactory.UserAccountTrie,
+			mrcf.numCrossShardPeers,
+			mrcf.numIntraShardPeers,
+			mrcf.numFullHistoryPeers,
+			i,
```

Comment:
please rename `i` to `idx` as to align with the rest of the file. I know. It's my code :D

Reply:
done